src/queryString/getQuery.js

```diff
@@ -4,7 +4,7 @@ define(function () {
      * Gets full query as string with all special chars decoded.
      */
     function getQuery(url) {
-        url = url.replace(/#.*/, ''); //removes hash (to avoid getting hash query)
+        url = url.replace(/#.*\?/, '?'); //removes hash (to avoid getting hash query)
```

Comment:
I think it should grab the first query string and ignore the second (if string contains 2).